### DISCUSSION TITLE
Mark media as owned

### DISCUSSION THREAD

**Discussion by orrrrn** [IgnisDa/ryot#489] [Ideas]
It would be great to be able to mark if you own media for movie/anime or whatever. This could then be used as a filter so one could browse genres, actors and everything using this "owned" status as a filter.

I could create collection for owned items but I can't use this as a "global" filter. This kind of filter would be enough for my use case.

I know that ryot is mostly for tracking activities instead of being catalog for what you have but this feature would expand possible use cases.

**Comment by IgnisDa** [comment]
Huh this is a really good idea. Can you make an issue for this? I'll implement it next week.
